x/gov/keeper/grpc_query_test.go

```diff
@@ -470,8 +470,9 @@ func (suite *KeeperTestSuite) TestGRPCQueryParams() {
 			func() {
 				req = &types.QueryParamsRequest{ParamsType: types.ParamVoting}
 				expRes = &types.QueryParamsResponse{
-					VotingParams: types.DefaultVotingParams(),
-					TallyParams:  types.NewTallyParams(sdk.NewDec(0), sdk.NewDec(0), sdk.NewDec(0)),
+					VotingParams:  types.DefaultVotingParams(),
+					DepositParams: types.DepositParams{MinInitialDepositRatio: sdk.ZeroDec()},
```

Comment:
Same question here about defaulting to zero for tests.

Reply:
That's because x/gov deposit param genesis value for this field is initialized to sdk.ZeroDec()